app/src/main/java/com/example/easycloset/AdditemActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.easycloset;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+public class AdditemActivity extends AppCompatActivity {
+
+    private Button btnCoats;
+    private Button btnJackets;
+    private Button btnSweaters;
+    private Button btnShirts;
+    private Button btnHoodies;
+    private Button btnShorts;
+    private  Button btnPants;
+    private  Button btnJoggers;
+    private Button btnBoots;
+    private Button btnSneakers;
+    private Button btnSlides;
+    private Button btnHeadwear;
+    private Button btnDone;
+
+
```

Comment:
Just general note. I will use prefix _nit_ for:
A minor complaint, of little importance. Often in the context of a code review. e.g. a typo in a comment, or a little extra whitespace.

nit: odd spacing; there should be a good reason to have more than one empty line

Reply:
fixed